src/gridfs/download.ts

```diff
@@ -184,6 +184,8 @@ export class GridFSBucketReadStream extends Readable implements NodeJS.ReadableS
    * the 'close' event once the cursor is successfully killed.
    *
    * @param callback - called when the cursor is successfully closed or an error occurred.
+   *
+   * @deprecated Callbacks are deprecated and will be removed in the next major version. See [mongodb-legacy](https://github.com/mongodb-js/nodejs-mongodb-legacy) for migration assistance
```

Comment:
you should make this match the way we do it in the upload abort, by first splitting off the overload, and then deprecating just the callback one, because we don't want to deprecate the entire method, just the callback version of it